scripts/run-if.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
Should this be `#!/bin/bash`? Supposedly that's what the bad substitution error we're getting in travis is from...

https://stackoverflow.com/a/23277604